### ISSUE TITLE
Feature Request: add Project activity under "File Shares"

### ISSUE THREAD

**Issue by sunjam** [nextcloud/activity#377]
### Steps to reproduce
1.  As of Nextcloud 16, share a file or folder by _[Add to a Project](https://nextcloud.com/blog/nextcloud-16-allows-you-to-link-resources-to-keep-track-of-your-projects/)_
![1558300312](https://user-images.githubusercontent.com/1787238/57988207-23e1b180-7a40-11e9-9fa7-764aebe3b6f8.png)
2.  Check Activity feed

### Expected behaviour
Project changes will display in the Activity Feed under _File Shares_
![1558300410](https://user-images.githubusercontent.com/1787238/57988224-5db2b800-7a40-11e9-88b7-d04c6f345187.png)
### Actual behaviour
Nothing is displayed in Activity app on Nextcloud 16.0.1

**Comment by nickvergessen** [comment]
cc @juliushaertl

**Comment by sunjam** [comment]
Just tried this again by attaching a file to a Deck board as a Project.  No activity is displayed. I thought something might pop up under the Activity categories for _Deck_ or _File Shares_.

**Comment by juliushaertl** [comment]
@nickvergessen makes sense to have activities for projects in general I guess, not just for files.
